rpcserver.go

```diff
@@ -1499,6 +1500,60 @@ func (r *rpcServer) PendingChannels(ctx context.Context,
 		}
 	}
 
+	// We'll also fetch all channels that are open, but have had their
+	// commitment broadcasted, meaning they are in the process of being
+	// closed.
+	openChannels, err := r.server.chanDB.FetchAllChannels()
+	if err != nil {
+		rpcsLog.Errorf("unable to fetch all channels: %v", err)
+		return nil, err
+	}
+
+	for _, pendingClose := range openChannels {
+
+		pub := pendingClose.IdentityPub.SerializeCompressed()
+		chanPoint := pendingClose.FundingOutpoint
+		channel := &lnrpc.PendingChannelsResponse_PendingChannel{
+			RemoteNodePub: hex.EncodeToString(pub),
+			ChannelPoint:  chanPoint.String(),
+			Capacity:      int64(pendingClose.Capacity),
+			LocalBalance:  int64(pendingClose.LocalCommitment.LocalBalance),
+		}
+
+		switch pendingClose.BroadcastedCommitment {
```

Comment:
Seems we also need to modify `ListChannels` to _not_ list any channels that have had their commitment transactions broadcast.

Reply:
Done.